templates/fluent-resource/en-US/security.ftl

```diff
@@ -0,0 +1,112 @@
+### Translation file for page: https://www.rust-lang.org/policies/security
+### templates/policies/security.hbs
+
+security-page-heading = Security policy
+
+security-reporting-heading = Reporting
+security-reporting-link = email security@rust-lang.org
+security-reporting-p1 =
+        Safety is one of the core principles of Rust, and to that end, we would
+        like to ensure that Rust has a secure implementation. Thank you for
+        taking the time to responsibly disclose any issues you find.
+security-reporting-p2 =
+        All security bugs in the Rust distribution should be reported by email
+        to <a href="mailto:security@rust-lang.org">security@rust-lang.org</a>.
```

Comment:
I would avoid having the email and the links for the security team to be changeable by translation. I see a slight vector there where a translation could be modified to say something else.

I'm not sure if I'm just paranoid there.

Same does, btw. apply to policy pages of all kind where core or team approval is necessary.

Reply:
I can definitely extract these links into [referencing messages](https://projectfluent.org/fluent/guide/references.html) for now. I'm not sure how to set them up as [variables](https://projectfluent.org/fluent/guide/variables.html) that get injected by the server, but that seems like a better way to do it in the long run.

> . . . a translation could be modified to say something else.

Isn't this a concern with all translation work though? I agree that variables at least gives us some assurances that typos weren't made by accident.

Reply:
@eignnx stick them in core.ftl for now. I'll move them out into a common file later.